osu.Game/Screens/Import/FileImportScreen.cs

```diff
@@ -0,0 +1,320 @@
+                                                    RelativeSizeAxes = Axes.X,
+                                                    AutoSizeAxes = Axes.Y,
+                                                    Spacing = new Vector2(10),
+                                                    Children = new Drawable[]
+                                                    {
+                                                        new SettingsEnumDropdown<FileFilterType>
+                                                        {
+                                                            Anchor = Anchor.BottomCentre,
+                                                            Origin = Anchor.BottomCentre,
+                                                            LabelText = "File Type",
+                                                            Current = filterType,
+                                                            Margin = new MarginPadding { Bottom = 15 }
+                                                        },
+                                                        new GridContainer
+                                                        {
+                                                            Anchor = Anchor.BottomCentre,
+                                                            Origin = Anchor.BottomCentre,
+                                                            AutoSizeAxes = Axes.Y,
+                                                            RelativeSizeAxes = Axes.X,
+                                                            Margin = new MarginPadding { Top = 15 },
+                                                            RowDimensions = new[]
+                                                            {
+                                                                new Dimension(GridSizeMode.AutoSize)
+                                                            },
+                                                            Content = new[]
+                                                            {
+                                                                new Drawable[]
+                                                                {
+                                                                    new TriangleButton
+                                                                    {
+                                                                        Anchor = Anchor.BottomCentre,
+                                                                        Origin = Anchor.BottomCentre,
+                                                                        RelativeSizeAxes = Axes.X,
+                                                                        Height = 50,
+                                                                        Width = 0.9f,
+                                                                        Text = "Refresh",
```

Comment:
I'd argue this just shouldn't exist. If we're adding a refresh button it would be part of the file chooser itself.